sdk/java/src/main/java/com/chain/api/CoreConfig.java

```diff
@@ -0,0 +1,141 @@
+package com.chain.api;
+
+import com.chain.http.*;
+import com.chain.exception.*;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.*;
+
+public class CoreConfig {
+
+  public static class Info {
+    public String state;
+
+    @SerializedName("is_configured")
+    public Boolean isConfigured;
+
+    @SerializedName("configured_at")
+    public Date configuredAt;
+
+    @SerializedName("is_signer")
+    public Boolean isSigner;
```

Comment:
Plus, it's a little easier on the consumer. The object types are easy to use incorrectly. Autoboxing can cause some really unexpected behavior, like a NullPointerException here:

```
if (config.isSigner == true) {
  // ...
}
```